utils/manifests/definitions.go

```diff
@@ -30,5 +30,10 @@ type CrdFilter struct {
 	GroupFilter   JsonPath //This will specify the path to get to group name
 	VersionFilter JsonPath //This will specify the path to get to version name. [Version should have a name field]
 	SpecFilter    JsonPath //This will specify the path to get spec
-	IsJson        bool     //Set to true if input format is json
+	ItrFilter     string   //Filter is appended with "=="<some-crd-name>")]", so no need to write it completely. Just write the part before double equals
+	ItrSpecFilter string   //Filter is appended with "=="<some-crd-name>")]", so no need to write it completely. Just write the part before double equals
+	VField        string
```

Comment:
Could you toss in a brief comment to describe these variables?

Reply:
Itr filter is for defintions
ItrSpec filter is for Schemas
The itr filter has to be postfixed by "==)]" and is used when we are filtering for certain specific crds or resources.
So we iterate over the entire json and extract only the objects we are interested in.

Reply:
@Revolyssup, I think @leecalcote meant to add some comments in the code. Right now the field names are too cryptic.  It would be come back to bite us when someone else would try to use the struct.

Reply:
Okay will do